installer.sh.in

```diff
@@ -1327,6 +1327,12 @@ ${BOLD}Do you want to continue?${RESET}" 20 80 || return
         unset USERLOGIN
     fi
 
+    POLKITRULE=$TARGETDIR/etc/polkit-1/rules.d/void-live.rules
+    if [ -f $POLKITRULE ]; then
+        rm -f $POLKITRULE
+    fi
```

Comment:
since you're using `rm -f`, you can skip the conditional directly. (and then also skip using a variable, I guess)